src/CachePlugin.php

```diff
@@ -138,6 +137,40 @@ public function handleRequest(RequestInterface $request, callable $next, callabl
     }
 
     /**
+     * Calculate the timestamp when this cache item should be dropped from the cache. The lowest value return after
+     * the calculation is $maxAge.
+     *
+     * @param int|null $maxAge
+     *
+     * @return int|null Unix system time. @see PSR6 for caching
```

Comment:
then lets replace the whole return to "This value is passed as cache lifetime to the PSR-6 cache". and not even talk about unix system time.

being private does not mean it does not need to be documented. i usually don't remember such things a year later even if i coded it myself, and here contributors might try to understand the code, so good documentation is valuable even on internal things.